zebrad/src/components/sync.rs

```diff
@@ -557,11 +557,11 @@ where
     fn update_metrics(&self) {
         metrics::gauge!(
             "sync.prospective_tips.len",
-            self.prospective_tips.len() as i64
+            self.prospective_tips.len() as f64
         );
         metrics::gauge!(
             "sync.downloads.in_flight",
-            self.downloads.in_flight() as i64
+            self.downloads.in_flight() as f64
```

Comment:
Is this change to floating-point required?

The underlying function has changed, and there are no issues with losing precision here. But it seems strange that it's required for these specific metrics, but not any others.
https://docs.rs/metrics/0.13.0-alpha.8/metrics/trait.Recorder.html#tymethod.update_gauge

Reply:
Yes, it doesn't compile otherwise.